### PR TITLE
Add support for team_reviewers & assignees

### PR DESCRIPTION
**Pull request by kuhnroyal**
This adds support for team_reviewers & assignees

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -23,12 +23,22 @@ async function run(): Promise<void> {
     const pullNumber = pullRequest.data.number;
     const htmlUrl = pullRequest.data.html_url;
 
-    if (reviewers.length > 0) {
+    if (reviewers.length > 0 || team_reviewers.length > 0) {
```
Comment (thomaseizinger) [inline src/index.ts:26]: Did you test that this is the required logic?

**Review by thomaseizinger — COMMENTED** [review]
Thanks and sorry for the long wait.

File: src/index.ts (hunk 1 above)
Comment (kuhnroyal) [reply src/index.ts:26]: I am currently using my fork in workflows and it works for both sets of reviewers.
Would love to switch back.

Comment (thomaseizinger) [reply src/index.ts:26]: Cool, thanks for the confirmation :)

**Comment by thomaseizinger** [comment]
A minor formatting problem, otherwise happy to merge this!

**Comment by kuhnroyal** [comment]
Can't commit those fixes because of the pre-commit hook. There are no changes to dist...

**Comment by kuhnroyal** [comment]
This should do it.

**Comment by thomaseizinger** [comment]
> Can't commit those fixes because of the pre-commit hook. There are no changes to dist...

Oh, I guess I outsmarted us there :(

Good to see you've found a way around it. I guess disabling the hooks for such a situation is always an option.

**Comment by thomaseizinger** [comment]
I'll kick out a release later.

**Comment by kuhnroyal** [comment]
Not sure why the integration tests failed but it seems unrelated (404)

**Comment by kuhnroyal** [comment]
@thomaseizinger Can you create a new release by any chance?

**Comment by thomaseizinger** [comment]
> @thomaseizinger Can you create a new release by any chance?

Done: https://github.com/thomaseizinger/create-pull-request/releases/tag/1.3.0

Sorry for the delay.

**Comment by kuhnroyal** [comment]
Thank you!

**Pull request merged**